src/Fluidity/Web/UI/App_Plugins/Fluidity/propertyeditors/entitypicker.controller.js

```diff
@@ -84,6 +84,11 @@
             angularHelper.getCurrentForm($scope).$setDirty();
         };
 
+        $scope.open = function (section, collection, id) {
+            var openHref = "/umbraco/#/" + section + "/fluidity/edit/" + collection + "!" + id;
+            window.location.href = openHref;
```

Comment:
Need to use the `$location` service and call `$location.path(/section/fluidity/edit/collection!id)`  instead of changing the window location explicitly as that's the angular way to do it. By changing it via window you are requiring the need of `/umbraco/` in the route which could break if someone changes this (it's possible to change in config).